TestClients/NICE.Identity.TestClient.NETFramework/Startup.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Configuration;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.Owin;
+using Microsoft.Owin.Security;
+using Microsoft.Owin.Security.Cookies;
+using Microsoft.Owin.Security.OpenIdConnect;
+using Newtonsoft.Json.Linq;
+using Owin;
+
+[assembly: OwinStartup(typeof(NICE.Identity.TestClient.NETFramework.Startup))]
+
+namespace NICE.Identity.TestClient.NETFramework
+{
+	public class Startup
+	{
+		public void Configuration(IAppBuilder app)
```

Comment:
I think this Startup class is doing too much - it would be a lot cleaner if the Auth0/OIDC stuff was moved to a separate ServiceCollection extension method (and class), and then we could use 'app.ConfigureAuth0' for example.

Edit - this will probably all be removed anyway once we inetgrate with the SDK I suppose